msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -846,20 +999,22 @@ private void acquireTokenSilent(@NonNull final String[] scopes,
 
         validateSilentParameters(acquireTokenSilentParameters);
 
-        acquireTokenSilentAsync(acquireTokenSilentParameters);
+        acquireTokenSilent(acquireTokenSilentParameters);
     }
 
     private void validateSilentParameters(
             @NonNull final AcquireTokenSilentParameters acquireTokenSilentParameters) {
+        /*
```

Comment:
Confused about the commented out code here. For the AcquireTokenSilentParameters, should we validate the authority and callback?